### PR TITLE
#751 updated benchmark pom and applied renaming

### PR DESCRIPTION
**Pull request by hmottestad**
This PR addresses GitHub issue: #751 .

Briefly describe the changes proposed in this PR:

* Fixed version number in the benchmark pom
* Applied the renaming of ForwardChainingSchemaCachingRDFSInferencer to the benchmarking classes

### CODE REVIEW

File: benchmark/pom.xml [hunk 1]
```diff
@@ -6,7 +6,7 @@
     <parent>
         <groupId>org.eclipse.rdf4j</groupId>
         <artifactId>rdf4j</artifactId>
-        <version>2.2-SNAPSHOT</version>
+        <version>2.3-SNAPSHOT</version>
```
Comment (jeenbroekstra) [inline benchmark/pom.xml]: This should have been automatically done by maven. Is the benchmark module not registered as a module in the root pom?

Comment (ansell) [inline benchmark/pom.xml]: It looks that way. Not a big issue that it wasn't released in 2.2 though.

**Comment by hmottestad** [comment]
Fixed the root pom

**Comment by jeenbroekstra** [comment]
I just realized that it may have been deliberate to keep the benchmark module out of the root pom, to avoid the benchmark tests automatically executing. I haven't verified that this actually happens (it may not as the benchmarks are run using a shell script I think, rather than a maven task) but we should doublecheck before merging.

**Pull request merged**